cluster/charts/crossplane/README.md

```diff
@@ -80,6 +80,7 @@ and their default values.
 | `customAnnotations` | Custom annotations to add to the Crossplane deployment and pod | `{}` |
 | `serviceAccount.customAnnotations` | Custom annotations to add to the serviceaccount of Crossplane | `{}` |
 | `priorityClassName` | Priority class name for Crossplane and RBAC Manager (if enabled) pods | `""` |
+| `hostNetwork` | Enable hostNetwork for Crossplane | `false` |
```

Comment:
@portswigger-tim if we add the ability to set this, it would be great to add a warning that this is giving Crossplane lots of privilege in a cluster and should be handled with caution.

Reply:
```suggestion
| `hostNetwork` | Enable hostNetwork for Crossplane. Caution: setting it to true means Crossplane's Pod will have high privileges. | `false` |
```